pkgs/top-level/all-packages.nix

```diff
@@ -31242,15 +31242,12 @@ in
   termpdfpy = python3Packages.callPackage ../applications/misc/termpdf.py {};
 
   inherit (callPackage ../applications/networking/cluster/terraform { })
-    terraform_0_12
-    terraform_0_13
-    terraform_0_14
-    terraform_0_15
```

Comment:
We just entirely drop them? In that case, we need to add a `throw` to the aliases, and add a changelog entry.